### ISSUE TITLE
omid_v1_present iframe missing sandbox attribute

### ISSUE THREAD

**Issue by bjoberg** [InteractiveAdvertisingBureau/Open-Measurement-JSClients#30]
# Overview
The iframe created to detect if omid is present, or not, does not contain a `sandbox` attribute. 

Current output:
```html
<iframe id="omid_v1_present" name="omid_v1_present" style="display: none;">
...
</iframe>
```

Desired output:
```html
<iframe id="omid_v1_present" name="omid_v1_present" style="display: none;" sandbox>
...
</iframe>
```

[Code reference](https://github.com/InteractiveAdvertisingBureau/Open-Measurement-JSClients/blob/9a7ff8be0c0a9e196bddbd4b0642583b910b7cfd/src/common/detect-omid.js#L54).

## Why is this a problem? 
This is a problem because the `sandbox` attribute "controls the restrictions applied to the content embedded in the <iframe>" ([ref](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox)). Although this iframe only exists so that verfication scripts can find where the OMSDK is, it is best practice to enable all iframe restrictions.

## What can be done?
Add `sandbox` to the iframe.

**Comment by nlehrer** [comment]
Yes, I think adding the `sandbox` attribute should be fine. See #31 for discussion of how it will be merged.

**Comment by nlehrer** [comment]
The change in the upstream repo was merged and will be included in v1.4.6 which is being released at the moment (should appear in this repo today or Monday).

**Comment by bjoberg** [comment]
Amazing 🙌🏼. Thanks for your support!!
